src/aspire/source/simulation.py

```diff
@@ -243,9 +243,10 @@ def projections(self):
         """
         return self._projections_accessor
 
-    def _projections(self, indices):
+    def _projections(self, indices, legacy=False):
```

Comment:
There are ways to avoid this extra argument; mentioned one in other remark.

If you keep the flag, Joakim can let you know if he prefers to continue using the other var name (zero_nyquist).

Reply:
Thought about using `zero_nyquist` too, but went with `legacy` so it would be more apparent that it would need to be removed if we eventually do away with `_LegacySimulation`.

Reply:
I figured, might not need. No worries.